playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -29,12 +29,18 @@
   - name: Check available disk space for etcd backup
     shell: df --output=avail -k {{ openshift.common.data_dir }} | tail -n 1
     register: avail_disk
+    # AUDIT:changed_when: `false` because we are only inspecting
```

Comment:
These audit notes are to explain why I added `changed_when` to tasks.

Because we are committed to performing a full `changed_when` audit in the future these notes are temporary. They also help me keep track of which items I have addressed in the play run results. We will eventually remove these comments, but for now I'm not very concerned about making a perfect audit syntax. The only thing that I am sticking to now is that all the audit annotations I am adding now follow these formats:

*Annotating a manually added override:*

```
# AUDIT:changed_when: <THE VALUE> because [EXPLANATION].
```

*Changes to tasks that don't explicitly override `changed_when` while we're getting the repo cleaned up:*

```
# AUDIT:changed_when_note: Testing <THE CHANGES YOU SEE>. <BRIEF EXPLANATION>.
```